translations/subsurface_source.ts

```diff
@@ -7258,6 +7258,186 @@ See https://doc.qt.io/archives/qt-4.8/qdatetime.html#fromString</source>
         <source>One can specify te relative position within the video from where the thumbnail is retrieved. Use the slider below do do that.</source>
         <translation type="unfinished"></translation>
     </message>
+    <message>
```

Comment:
The translations are extracted automatically, so this part is not needed (and might interfere with adding the generated translations).

Reply:
Alright, I have reverted the changes.